cloud_functions/token_refresh/main.py

```
@@ -0,0 +1,65 @@
+import functions_framework
+from google.cloud import firestore
+from firebase_admin import auth, initialize_app
+
+from .streams.strava import strava_refresh_oauth_token
+from .utils.logger_utils import logger
+
+# Initialize Firebase Admin
+initialize_app()
+
+@functions_framework.http
+def refresh_token(request):
```

Ruff (0.8.2)

`refresh_token` is too complex (12 > 10)

(C901)